tests/garage/tf/policies/test_deterministic_mlp_policy_with_model.py

```diff
@@ -0,0 +1,66 @@
+import pickle
+from unittest import mock
+
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.envs import TfEnv
+from garage.tf.models import Model
+from garage.tf.policies import DeterministicMLPPolicyWithModel
+from tests.fixtures import TfGraphTestCase
+from tests.fixtures.envs.dummy import DummyBoxEnv
+
+
+class SimpleMLPModel(Model):
```

Comment:
if you are going to use a fixture repeatedly, please put it in tests/fixtures rather than copying it